orchagent/portsorch.cpp

```diff
@@ -498,6 +502,39 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
         }
     }
 
+    // Query whether SAI supports Host Tx Signal and Host Tx Notification
+
+    sai_attr_capability_t capability;
+    if (sai_query_attribute_capability(gSwitchId, SAI_OBJECT_TYPE_PORT,
+                                            SAI_PORT_ATTR_HOST_TX_SIGNAL_ENABLE,
+                                            &capability) == SAI_STATUS_SUCCESS)
+    {
+        if (capability.create_implemented == true)
+        {
+            SWSS_LOG_DEBUG("SAI_PORT_ATTR_HOST_TX_SIGNAL_ENABLE is true");
+            m_saiHwTxSignalSupported = true;
+        }
+    }
+
+    if (sai_query_attribute_capability(gSwitchId, SAI_OBJECT_TYPE_SWITCH,
+                                            SAI_SWITCH_ATTR_PORT_HOST_TX_READY_NOTIFY,
+                                            &capability) == SAI_STATUS_SUCCESS)
+    {
+        if (capability.create_implemented == true)
+        {
+            SWSS_LOG_DEBUG("SAI_SWITCH_ATTR_PORT_HOST_TX_READY_NOTIFY is true");
+            m_saiTxReadyNotifySupported = true;
+        }
+    }
+
+    if (m_saiHwTxSignalSupported && m_saiTxReadyNotifySupported)
+    {
+        SWSS_LOG_DEBUG("m_cmisModuleAsicSyncSupported is true");
+        m_cmisModuleAsicSyncSupported = true;
+        auto transceiverInfoTableName = STATE_TRANSCEIVER_INFO_TABLE_NAME;
```

Comment:
this Variable is not needed, please remove it and use STATE_TRANSCEIVER_INFO_TABLE_NAME directly